### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by drewm8080**
Sorry. Forgot to fix something in the last pull request. Use this one.

### CODE REVIEW

**Comment by c-myau** [comment]
Just write a new commit to the pull request next time. You don't need to open a new PR with every edit.

File: timeserieskmeans_final.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  plt.rcParams['legend.fontsize'] = 12 
+  plt.rcParams['figure.titlesize'] = 12 
+  plt.rcParams['image.cmap'] = 'jet' 
+  plt.rcParams['image.interpolation'] = 'none' 
+  plt.rcParams['figure.figsize'] = (12, 10) 
+  plt.rcParams['axes.grid']=True
+  plt.rcParams['lines.linewidth'] = 2 
+  plt.rcParams['lines.markersize'] = 8
+  data_c = df.copy()
+  data_c['Class']=y
```
Comment (c-myau) [inline timeserieskmeans_final.py:64]: If we have this many parameters, pass in a JSON object or a dictionary or something into the function and define these at script runtime.

File: timeserieskmeans_final.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+from tslearn.clustering import TimeSeriesKMeans
+
+def kmeans(csv,k,k_seeds,xaxis, yaxis, x_lower=None,x_upper=None,y_lower=None,y_upper=None):
+  df = pd.read_csv(f'{csv}')
+  df = df.dropna()
+
+  df['Day'] = df.index +1
+
+
+  #assuming first column is the datetime column
```
Comment (c-myau) [inline timeserieskmeans_final.py:38]: Jae has an example CSV on google drive but regardless, all we are doing is importing the CSV directly into a dataframe with read_csv(). Use the timeseries column from that

File: timeserieskmeans_final.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    plt.xlim(x_lower, x_upper)
+  elif y_lower and y_upper is not None:
+    plt.ylim(y_lower,y_upper)
+
+
+  sns.scatterplot(x=f"{xaxis}",y=f'{yaxis}',hue='Class',data=data_c,palette='plasma')
+  plt.scatter(centroids[:,0] , centroids[:,1] , s = 250, color = 'k',alpha = 0.9)
+
+  dtw_score = dtw(x, y)
+  print("The DTW score is", dtw_score)
```
Comment (c-myau) [inline timeserieskmeans_final.py]: Include a [if __name__ == __main__ ... section along with the sample csv data used so we can test this script out as a standalone.

File: timeserieskmeans_final.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+from tslearn.metrics import dtw
+from tslearn.metrics import soft_dtw
+from tslearn.clustering import TimeSeriesKMeans
+
+def kmeans(csv,k,k_seeds,xaxis, yaxis, x_lower=None,x_upper=None,y_lower=None,y_upper=None):
```
Comment (c-myau) [inline timeserieskmeans_final.py]: Where is the scale parameter for X or Y? We need to be able to adjust the magnitude of the X or Y axis at runtime. Ideally, pass in a mathematical function as a parameter.

**Comment by drewm8080** [comment]
Screen Shot 2022-11-11 at 5 10 42 PM


For the most recent file, this was the output. I used a standard scaler on both the x and y axis.

**Review by c-myau — APPROVED** [review]
Some minor tweaks could be needed but otherwise LGTM

**Pull request merged**